src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -557,8 +567,8 @@ public void AnalyzeCommand_DefaultEndToEndAnalysis()
                 (toolNotification) => { toolNotificationCount++; },
                 (configurationNotification) => { configurationNotificationCount++; });
 
-            // By default, the exception raising rule produces a single error.
-            // The simple test rule doesn't raise anything without add'l configuration
+            // As configured by injected TestRuleBehaviors, we should
+            // see an error per scan target (one file in this case).
             resultCount.Should().Be(1);
             run.Results[0].Kind.Should().Equals(ResultKind.NotApplicable);
```

Comment:
>Equals [](start = 41, length = 6)

LOL. `Equals` => `Be`. As written this code compares the `Should` method to the `NotApplicable` constant and discards the result :-)

Reply:
(existing bug)

---
In reply to: [322900451](https://github.com/microsoft/sarif-sdk/pull/1670#discussion_r322900451) [](ancestors = 322900451)